webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -551,7 +552,7 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
           break;
         case ExperimentAction.Edit:
           if (data) updateExperiment(data);
-          notification.success({ message: 'Experiment updated successfully' });
+          openToast({ severity: 'Confirm', title: 'Experiment updated successfully' });
```

Comment:
whats the difference between notification and openToast?

Reply:
Via GQ: "openToast is a style wrapper over notification.open  and we should prefer using openToast"